src/interfaces/continent.ts

```diff
@@ -1,84 +1,66 @@
+  readonly map_pos: [number, number]; // Map coordinate of the base icon & name
+  readonly type_id: number;
+  readonly type_name: string;
+  readonly resource_amount: number;
+  readonly resource_id?: number;
+  readonly resource_name?: string;
 }
 
-export interface ContinentView {
-  view_url: string;
-  details: ContinentDetails;
+export interface BaseStatus {
+  readonly id: number;
+  readonly server_id: number;
+  readonly population: Population;
+  readonly owning_faction?: number;
+  readonly owning_outfit?: number;
+  readonly held_since: number; // UTC timestamp of the last cap or reset
 }
 
-export type ContinentViews = Array<ContinentView>;
-export type Continents = Array<ContinentDetails>;
-
 /**
- * Convert a list of `ContinentDetails` to `ContinentView`
- * @param conts The ContinentDetails to add view URLs to
- * @param view_urls The URLs to add to `conts`
- * @throws The number of URLs does not match the number of continents
+ * Static continent data, loaded once and cached.
  */
-export const to_continent_view = (
-  conts: Array<ContinentDetails>,
-  view_urls: Array<string>
-): ContinentViews => {
-  if (conts.length !== view_urls.length)
-    throw new Error(
-      "The number of URLs does not match the number of continents"
-    );
-  return conts.map((cont, i) => ({ details: cont, view_url: view_urls[i] }));
-};
-
-export type faction = "NC" | "TR" | "VS" | "NS";
+export interface ContinentInfo {
+  readonly id: number;
+  readonly name: string;
+  readonly description: string;
+  readonly lattice_links: Array<[number, number]>; // Numbers are base IDs
+  readonly map_outlines_svg: BaseSvgMapping;
+  readonly map_tileset: string;
+}
 
 /**
- * Represents a base.
+ * Dynamic continent update
  */
-export interface BaseState {
-  readonly name: string;
-  readonly id: string;
-  /**
-   * Should be one of the following: [NC, TR, VS, NS]
-   * Neutral is NS
-   */
-  readonly faction: faction;
-  readonly continent_id: number;
-  /**
-   * How long since it was last capped in ms
-   */
-  readonly time_held: number;
-  readonly captured_by: null | Outfit;
+export interface ContinentStatus {
+  readonly id: number;
+  readonly server_id: number;
   readonly population: Population;
-  /**
-   * This is the priority level assigned by the AI.
-   */
-  readonly priority_level: number;
+  readonly status: string;
+  readonly locked_by?: number;
+  readonly alert_active: boolean;
+  readonly alert_started?: number;
+  readonly alert_ends?: number;
 }
 
-export type ContinentGetter = ApiGetter<ContinentDetails>;
+/**
+ * Faction-specific population data.
+ */
+export interface Population {
+  readonly NC: number;
+  readonly TR: number;
+  readonly VS: number;
+  readonly NSO: number;
+}
 
 /**
- * A more readable representation of the continent IDs
+ * Mapping of string base IDs to their outline SVG.
  */
-export enum ContinentIds {
-  INDAR = 2,
-  HOSSIN = 4,
-  AMERISH = 6,
-  ESAMIR = 8,
+export interface BaseSvgMapping {
```

Comment:
Not sure if this needs a standalone interface... it's a JSON object mapping stringified IDs to serialised SVGs; that could probably be inline as well.